.actions.sh

```diff
@@ -5,14 +5,14 @@ echo "Running sinatra tests..."
 bundle exec rake
 
 echo "Running sinatra-contrib tests..."
-export BUILDIR=$TRAVIS_BUILD_DIR/sinatra-contrib
+export BUILDIR=$GITHUB_WORKSPACE/sinatra-contrib
```

Comment:
require 'sinatra'
    get '/frank-says' do
      'Put this in your pipe & smoke it!'
    end